frontend/src/container/Trace/Search/AllTags/Tag/index.tsx

```diff
@@ -16,16 +17,79 @@ type Tags = FlatArray<TraceReducer['selectedTags'], 1>['Operator'];
 interface AllMenuProps {
 	key: Tags | '';
 	value: string;
+	supportedTypes: string[];
 }
 
-const AllMenu: AllMenuProps[] = [
+export const AllMenu: AllMenuProps[] = [
 	{
-		key: 'in',
+		key: 'In',
 		value: 'IN',
+		supportedTypes: ['string'],
 	},
 	{
-		key: 'not in',
+		key: 'NotIn',
 		value: 'NOT IN',
+		supportedTypes: ['string'],
+	},
+	{
+		key: 'Equals',
+		value: 'EQUALS',
+		supportedTypes: ['string', 'number', 'boolean'],
```

Comment:
can we keep a common `['string', 'number', 'boolean']` so that we don't repeat our selves